test/output-tests/run_ml.expected

```diff
@@ -9,925 +19,99 @@ run_ml.at-122-prog.cob:21.14-21.16:
-  82 >            XML GENERATE out FROM 0specialtagname
-----                                                   ^
-  83              IF out <> '<_0SpecialTAGName>abc</_0SpecialTAGName>'
-  84                  DISPLAY 'Test 9 failed: ' FUNCTION TRIM (out)
->> Hint: Missing .
-
-run_ml.at-122-prog.cob:83.11-83.13:
-  80   
-  81              *> Test the case of the id in the DATA DIVISION is preserved.
-  82              XML GENERATE out FROM 0specialtagname
-  83 >            IF out <> '<_0SpecialTAGName>abc</_0SpecialTAGName>'
+run_ml.at-122-prog.cob:23.11-23.13:
+  20          01  employee.
+  21              05 id            PIC 9(1)  value 1.
+  22              05 name          PIC X(10) value "Someone".
+  23 >            05 dept          PIC X(10) value "Marketing".
 ----              ^^
-  84                  DISPLAY 'Test 9 failed: ' FUNCTION TRIM (out)
-  85              END-IF
->> Warning: Invalid syntax
-
-run_ml.at-122-prog.cob:89.11-89.14:
-  86              .
-  87   
-  88              *> Another test with mixed attributes and values
-  89 >            XML GENERATE out FROM EMPLOYEE TYPE OF ID IS ATTRIBUTE
-----              ^^^
-  90              IF out <> '<employee id="1">'-
-  91                        '<name>Someone</name>'-
+  24   
+  25          PROCEDURE            DIVISION.
 >> Error: Invalid syntax
 
-run_ml.at-122-prog.cob:89.27:
-  86              .
-  87   
-  88              *> Another test with mixed attributes and values
-  89 >            XML GENERATE out FROM EMPLOYEE TYPE OF ID IS ATTRIBUTE
-----                              ^
-  90              IF out <> '<employee id="1">'-
-  91                        '<name>Someone</name>'-
->> Hint: Missing .
-
-run_ml.at-122-prog.cob:89.28-89.32:
+run_ml.at-122-prog.cob:89.52:
   86              .
   87   
   88              *> Another test with mixed attributes and values
   89 >            XML GENERATE out FROM EMPLOYEE TYPE OF ID IS ATTRIBUTE
-----                               ^^^^
-  90              IF out <> '<employee id="1">'-
-  91                        '<name>Someone</name>'-
->> Warning: Invalid syntax
-
-run_ml.at-122-prog.cob:89.41:
-  86              .
-  87   
-  88              *> Another test with mixed attributes and values
-  89 >            XML GENERATE out FROM EMPLOYEE TYPE OF ID IS ATTRIBUTE
-----                                            ^
+----                                                       ^
   90              IF out <> '<employee id="1">'-
   91                        '<name>Someone</name>'-
->> Hint: Missing .
+>> Hint: Missing DIVISION
 
-run_ml.at-122-prog.cob:89.42-89.46:
+run_ml.at-122-prog.cob:89.53-89.55:
   86              .
   87   
   88              *> Another test with mixed attributes and values
   89 >            XML GENERATE out FROM EMPLOYEE TYPE OF ID IS ATTRIBUTE
-----                                             ^^^^
+----                                                        ^^
```

Comment:
This also seems like an unwanted new error

Reply:
Same reason as above.